Update readme with new process

WaleedAshraf opened the pull request:
ping: @nodejs/community-committee

@RRomoff can you please go through the "Review Process for Original Posts" and "Review Process for Created Medium Posts" sections?
I think we can add more details and make it easier to understand but couldn't find time to do that.

We also need to to add a link for Trello.

closes https://github.com/nodejs/community-committee/issues/439 https://github.com/nodejs/community-committee/issues/491

WaleedAshraf commented:
ping @nodejs/community-committee 
Kindly review this.

keywordnew commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
+
+### Key Roles:
+- **Contributor/Writer** : Person submitting article or idea for inclusion on the Node.js Collection
+- **Editor** : Technical Reviewer. Person who ensure the content is technically sound/ appropriate.
```
```suggestion
- **Editor** : Technical Reviewer. Person who ensures the content's technical integrity.
```

WaleedAshraf commented:
Ping @nodejs/community-committee @amiller-gh @bnb 
If possible, please review this before the meeting tomorrow.

WaleedAshraf commented:
ping @nodejs/community-committee, please review

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
 If you're passionate about something not listed above, please don't hesitate to submit your idea for consideration.
 
+## Resources
+Trello Board [https://trello.com/b/M9VUQl9S/nodejs-collection](https://trello.com/b/M9VUQl9S/nodejs-collection)
```
Board not found.

WaleedAshraf replied:
@RRomoff to make it public

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
-- Posts from members and community members must be vendor neutral, though they may mention vendors involved in specific project deployments if this serves the purpose of educating readers. 
-- Posts about hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, are allowed, but again shouldn’t feel like an advertisement for your services or company. 
+- Posts from members and community members must be vendor-neutral, though they may mention vendors involved in specific project deployments if this serves the purpose of educating readers.
+- Posts about hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, are allowed, but again shouldn’t feel like an advertisement for your services or company.
```
I'm not sure if this is actually valuable in practice. Have we had any posts that are like this?

WaleedAshraf replied:
@bnb I didn't get what you mean? Can you explain a bit?
To me, "Advertisement for your services or company" is not the best. So, it'd be good to keep this line.

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
-- Node.js Foundation Medium posts should be no longer than 1,000 words and no shorter than 300 words. 
-
-Diagrams or photos are strongly encouraged. 
+- When showing upstreaming of a patch fixing an issue for others, link back to the Github issue so readers can follow along.
```
Is there a case where this line has actually been used? If not, perhaps worth stiriking it entirely?

Additionally, if we don't strike it, it should be updated to `GitHub`

WaleedAshraf replied:
I'll fix it as `GitHub`

bnb replied:
This was resolved but not fixed 😅

WaleedAshraf replied:
Done

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
-Diagrams or photos are strongly encouraged. 
+- When showing upstreaming of a patch fixing an issue for others, link back to the Github issue so readers can follow along.
+- We don’t avoid critical commentary or broad issues but approach them with sensitivity, professionalism, and tact in a way that is beneficial and positive for the community.
+- Node.js Foundation Medium posts should be no longer than 1,000 words and no shorter than 300 words.
```
```suggestion
- Node.js Collection posts should be no longer than 1,000 words and no shorter than 300 words.
```

WaleedAshraf replied:
👍

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
-- If you are submitting an article or presentation that already exists, please send it in its entirety with a note on the expressed permission to publish from the owner of content.
-- Once your submission has been approved, it will be added to the Node.js Foundation Medium publication publishing calendar and a publish date will be provided, so you may plan to promote accordingly through your personal and company’s social media channels. 
+- Please submit a brief summary and the topic of the post to `medium@nodejs.org` for consideration.
+- Members of the Node.js Collection Technical Review Team and Node.js ProjectPR team will review your submission in a timely manner and provide the green light to draft the entire article or provide feedback or direction.
```
AFAIK this should be:

```suggestion
- Members of the Node.js Collection Technical Review Team and OpenJS Foundation PR team will review your submission in a timely manner and provide the green light to draft the entire article or provide feedback or direction.
```

WaleedAshraf replied:
👍

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
+
+- Editors have one week to provide approval or feedback - this is done internally on the alias and not to be shared with the writer until a final decision and suggested changes are reached as a group.
+
+- If the abstract is approved, the writer will be informed and a task will be created on the [Trello board](https://trello.com/b/M9VUQl9S/nodejs-collection) for the post
```
404 Trello board and missing period at the end.

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
+
+- The contributor will be made a “Writer” on the Node.js Collection medium account.
+
+- The contributor will write the first draft and Write will submit this draft on Medium and an editor or two will be assigned to review. The contributor will ping the medium alias to alert the editors a draft has been submitted. Clear directions on how to submit will be provided to the contributor.
```
> The contributor will write the first draft and Write will 

Not quite correctly phrased. `Write` is an error.

WaleedAshraf replied:
-> writer

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
+
+- The contributor will write the first draft and Write will submit this draft on Medium and an editor or two will be assigned to review. The contributor will ping the medium alias to alert the editors a draft has been submitted. Clear directions on how to submit will be provided to the contributor.
+
+- Editor from the Node.js Collection will review the post and provide the writer comments or feedback. This process can take up to a month to complete.
```
How is this substantially different from the 1 week period stated on line 57?

WaleedAshraf replied:
Line 57 is first approval if it is good enough to include or not.
If it is approved, then we start the review process, which can take up to a month.

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
+
+- Once the piece is complete, and the writer approves the content, an editor from Node.js PR team to review one final time and accept into publication.
+
+- Once the submission has been approved, it will be added to the Node.js Collection Medium publication - a publication date will be provided for proper social planning/ promo.
```
Additional unnecessary space after the slash in this line (`planning/ promo.`)

bnb replied:
```suggestion
- Once the submission has been approved, it will be added to the Node.js Collection Medium publication - a publication date will be provided for proper social planning/promo.
```

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
+
+## Review Process for Existing Created Medium Posts (to be republished with permission on the Node.js Collection)
+
+- Members from the evangelism working group are invited to become editors on Medium. If interested, contact here: medium@nodejs.org
```
There is no such thing as the Evangelism WG. This line should likely be deleted, as it doesn't seem to be relevant to the heading.

WaleedAshraf replied:
I will update it as "Anyone is invited to ......"
Later, we can limit it to CommComm and TSC if needed.

Ok?

bnb replied:
SGTM

bnb commented on README.md:
```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
+
+- The editors decide if the content is acceptable for publication.
+
+- If accepted, a task will be created in the [Trello board](https://trello.com/b/M9VUQl9S/nodejs-collection) and an editor will be assigned to review the content.
```
404 Trello Board.

bnb commented:
FWIW I would like to review once the above changes are made. Currently just pending those changes.

bnb commented on README.md:
```diff
@@ -24,39 +24,74 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
 If you're passionate about something not listed above, please don't hesitate to submit your idea for consideration.
 
+## Resources
+Trello Board [https://trello.com/b/M9VUQl9S/nodejs-collection](https://trello.com/b/M9VUQl9S/nodejs-collection)
```
```suggestion

* [Trello Board](https://trello.com/b/M9VUQl9S/nodejs-collection)
```

WaleedAshraf commented:
@nodejs/community-committee 
I have added
- Editors
- How to become an editor?
- Benefits of being an editor
- Current Editors

and removed the specific details about the submission process. There is still info available about the basic process and I think that is enough. We don't need to overdo it.
Let's discuss this in the next meeting.

bnb commented on README.md:
```diff
@@ -11,6 +11,23 @@ The Node.js Foundation’s new [Medium](https://medium.com/) publication serves
+We have list of people who review the post before publishing. If the post is specific to some technical topic, we seek review from someone who has knowledge of that domain.
+
+### How to become an editor?
+We are looking for more reviewers. If you are interested, please open an issue in [nodejs-collection repo](https://github.com/nodejs/nodejs-collection/issues) with your details and the team will review it.
```
What information needs to be included to be an editor?

WaleedAshraf replied:
I think it should include,
- Intro about themself.
- How much time they can give.
- Area of expertise.
- Email address for contact.

What do you suggest?

bnb commented on README.md:
```diff
@@ -15,13 +15,15 @@ The Node.js Collection is a [Medium](https://medium.com/) publication which serv
+We are looking for more reviewers. If you are interested, please open an issue in [nodejs-collection repo](https://github.com/nodejs/nodejs-collection/issues) with these details and the team will review it.
+- Name
+- Experties (i.e IoT, Node.js Modules, cli, APIs etc)
+- How much time you can give per week. (i.e 1-2 hours)
```
```suggestion
Please include the following:
- Name
- Expertise (i.e IoT, Node.js Modules, CLI, APIs etc)
- How much time you can give per week (i.e 1-2 hours)
```

bnb commented on README.md:
```diff
@@ -15,13 +15,15 @@ The Node.js Collection is a [Medium](https://medium.com/) publication which serv
 - You'll be added to [editors list on Medium](https://medium.com/the-node-js-collection/about).
-- You'll receive discount to attend OpenJS Foundation Conference.
-
+- Possibility to become a member of Node.js Project and avail benefits like [Travel Fund](https://github.com/nodejs/admin#travel-fund).
```
This is a benefit of being a member of the project.

If being an editor includes being a member of the project, this is inherently true and phrasing it like this is slightly misleading since it's something that comes inherently anyway rather than a specific benefit of this work.

If being an editor does not include being a member of the project, then this is also slightly misleading since there is additional work above and beyond being an editor that this requires.

WaleedAshraf replied:
Being an editor will not include being project member. I think we should keep it nomination based. Like [here](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#collaborator-nominations).
Once we see, someone is actively reviewing posts for 2,3 months, we can nominate them to become part of Node.js Collection ( -> Community Committee -> Node.js Project).

I also don't want to overdo it in the start. Let's go with an easy and simple process, once we have enough editors, we can even put this on hold.

Point to mention Travel Fund here was that people know what benefits would they get in return. People who are not part of the project already, may not even know that there is a travel fund benefit.

WaleedAshraf replied:
In the next PR, I'll introduce a custom logger as discussed in slack. For this PR, ignore it.

bnb replied:
>Point to mention Travel Fund here was that people know what benefits would they get in return.

This is a good thing to include for the relevant groups. I do think it's important though to include it in the correct place, and, as you said, this is not something that people _will_ get but something they _could_ get if they stick around. 

>Once we see, someone is actively reviewing posts for 2,3 months, we can nominate them to become part of Node.js Collection ( -> Community Committee -> Node.js Project).

Is this documented anywhere? If so, I think that is the most appropriate place to include that context since it's the context in which it is accurate.

bnb commented on README.md:
```diff
@@ -18,7 +18,8 @@ We have list of people who review the post before publishing. If the post is spe
 - Experties (i.e IoT, Node.js Modules, cli, APIs etc)
-- How much time you can give per week. (i.e 1-2 hours)
+- How much time you can give per week (i.e 1-2 hours)
+- Email address
```
```suggestion
Please include the following:
- Name
- Expertise (i.e IoT, Node.js Modules, CLI, APIs etc)
- How much time you can give per week (i.e 1-2 hours)
- Email address
```

WaleedAshraf replied:
Done.

WaleedAshraf commented:
@nodejs/community-committee It seems I would not be able to find time for this any soon. I'd be happy to help if someone else takes this from here.
Let's talk about this at the next meeting.

WaleedAshraf commented:
Closing in favor of https://github.com/nodejs/nodejs-collection/pull/41
